packages/base/field-component.gts

```diff
@@ -89,13 +90,19 @@ export function getBoxComponent(
     };
   }
 
-  let component: TemplateOnlyComponent<{ Args: { format?: Format } }> =
-    <template>
-      {{#let (lookupFormat @format) as |f|}}
+  let component: TemplateOnlyComponent<{
+    Args: { format?: Format; displayContainer?: boolean };
+  }> = <template>
+    {{#let (lookupFormat @format) as |f|}}
+      {{#let
```

Comment:
You can yield 2 vars in a single {{#let}} instead of nesting them.  there are a couple examples in the code base of this if you search around.

Reply:
Fixed now, thanks.